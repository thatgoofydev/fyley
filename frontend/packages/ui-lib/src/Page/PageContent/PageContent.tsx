import React, { FunctionComponent } from "react";
import styles from "./PageContent.module.scss";

export const PageContent: FunctionComponent = ({ children }) => {
  return <div className={styles.pageContent}>{children}</div>;
};
