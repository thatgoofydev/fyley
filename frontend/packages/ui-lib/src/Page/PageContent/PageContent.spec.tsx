import React from "react";
import { render, cleanup } from "@testing-library/react";
import { PageContent } from "./PageContent";

afterEach(cleanup);

describe("PageContent", function () {
  it("should render", function () {
    const { container, getByText } = render(
      <PageContent>
        <p>Some content</p>
      </PageContent>
    );

    expect(getByText("Some content")).not.toBeNull();
    expect(container).toMatchSnapshot();
  });
});
