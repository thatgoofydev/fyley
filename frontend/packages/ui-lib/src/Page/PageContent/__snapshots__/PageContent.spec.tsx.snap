// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent should render 1`] = `
<div>
  <div
    class="pageContent"
  >
    <p>
      Some content
    </p>
  </div>
</div>
`;
